stan/math/prim/prob/wiener5_lpdf.hpp

```diff
@@ -241,19 +351,47 @@ double grad_wiener5_a(double y, double a, double vn, double wn, double sv,
   return WrtLog ? ans * exp(ld) : ans;
 }
 
+/**
+ * Calculate the derivative of the wiener5 density w.r.t. 'v'
+ *
+ * @tparam WrtLog Whether to return the derivative w.r.t.
+ *                  the natural or log-scale density
+ *
+ * @param y A scalar variable; the reaction time in seconds
+ * @param a The boundary separation
+ * @param vn The relative starting point
+ * @param wn The drift rate
+ * @param sv The inter-trial variability of the drift rate
+ * @param err The log error tolerance
+ * @return Gradient w.r.t. v
+*/
 template <bool WrtLog = false>
-double grad_wiener5_v(double y, double a, double vn, double wn, double sv,
-                      double err = log(1e-12)) {
+inline double grad_wiener5_v(double y, double a, double vn, double wn,
+                              double sv, double err = log(1e-12)) {
```

Comment:
If this cannot throw should it be `noexcept`?

Reply:
Yes.